extensions/wikia/AdEngine/js/template/interstitial.js

```diff
@@ -0,0 +1,60 @@
+/*global define*/
+define('ext.wikia.adEngine.template.interstitial', [
+	'wikia.document',
+	'wikia.log',
+	'wikia.ui.factory',
+	'wikia.window'
+], function (doc, log, uiFactory, win) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.template.interstitial',
+		modalId = 'ext-wikia-adEngine-template-interstitial',
+		modalPadding = 20,
+		pollingInterval = 1000; // 1s
+
+	function show(params) {
+		log(['show', params], 'debug', logGroup);
+
+		var iframe = doc.getElementById(params.slot).querySelector('iframe'),
+			iframeBody = iframe.contentWindow.document.body,
+			selector = params.elem;
+
+		function onFound(element) {
```

Comment:
I'd prefer to define these functions outside of the `show` function so that they don't need to be redefined every time show is called. Looks like you could pass `params` to `onFound` or set `params` to a local variable scoped outside of the `show` function in order to pass that value around.